pkg/server/server.go

```diff
@@ -291,6 +299,45 @@ func WithMaxAuthorizationModelSizeInBytes(size int) OpenFGAServiceV1Option {
 	}
 }
 
+// WithDispatchThrottlingCheckResolverEnabled sets whether dispatch throttling is enabled.
+// Enable this feature will prioritize processing simple check and list objects requests
```

Comment:
Enabling this feature will prioritize dispatched requests requiring less than the configured dispatch level threshold over requests whose dispatch count exceeds the configured level.